fhir-database-utils/src/test/java/com/ibm/fhir/database/utils/query/WhereFragmentTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query;
+
+import static com.ibm.fhir.database.utils.query.expression.ExpressionUtils.bind;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.database.utils.query.expression.StringExpNodeVisitor;
+import com.ibm.fhir.database.utils.query.node.ExpNode;
+
+/**
+ * Unit tests for the {@link WhereFragment} class.
+ */
+public class WhereFragmentTest {
+
+    @Test
+    public void likeTest() {
+        WhereFragment where = new WhereFragment();
+        where.col("tab", "foo").like(bind("hello%")).escape("+");
+
+        ExpNode filter = where.getExpression();
+        StringExpNodeVisitor visitor = new StringExpNodeVisitor();
+        String expression = filter.visit(visitor);
+        assertEquals(expression, "tab.foo LIKE ? ESCAPE '+'");
+    }
+}
```

Comment:
I think we  should (as a team)  each take one of these test classes and work an issue on raising test  coverage so we can  get use to  the new pattern

Reply:
good idea

Reply:
#2319